source/initial_composition/world_builder.cc

```diff
@@ -49,9 +49,9 @@ namespace aspect
   {
     ASPECT_REGISTER_INITIAL_COMPOSITION_MODEL(WorldBuilder,
                                               "world builder",
-                                              "Specify the composition in through the World Buider. "
+                                              "Specify the initial composition through the World Builder. "
                                               "More information on the World Builder can be found at "
-                                              "https://geodynamicworldbuilder.github.io ."
+                                              "https://geodynamicworldbuilder.github.io. "
```

Comment:
the space after "io" is probably there so that is it not seen as part of the url. How are we handling URLs in parameters, anyway? With or without ``{url}``?

Reply:
I do not think there is a policy yet, but including links is reasonable. I have checked that `\url` works in the pdf, and included it.